src/components/modal-compound/AddFolderModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ChangeEvent, useState } from "react"
+import { Modal } from "./context/ModalContext"
+
+interface AddFolderModalProps {
+  onCloseModal: () => void
+}
+
+export default function AddFolderModal({ onCloseModal }: AddFolderModalProps) {
+  const [inputValue, setInputValue] = useState("")
+  const handleChangeInputValue = (event: ChangeEvent<HTMLInputElement>) => setInputValue(event.target.value)
```

Comment:
```suggestion
  const handleChangeInputValue: ChangeEventHandler<HTMLInputElement> = (event) => setInputValue(event.target.value)
```

함수 표현식을 쓰면 변수에 바로 타입을 바인딩할 수 있어서 타입 유추가 편하고 함수 파라미터 부분을 깔끔하게 관리할 수 있습니다.

이런식으로 사용해보시는건 어떨까요?

Reply:
와 .. 이 방법은 전혀 모르고있었습니다. 감사합니다